cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("connecting to db")
+        connection = None
+        while not connection:
```

Comment:
The variable `connection` is initially set to `None` and then checked in a `while not connection` loop. However, the `connection` variable is assigned a connection object in the `try` block, which will always evaluate to `True` even if the connection is not successful. Consider using a flag variable to control the loop instead.